modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/hooks/useGetActivationStatusCardLayout.js

```diff
@@ -0,0 +1,134 @@
+		subscriptionGroupLxcEnvironment?.activationStatus
+	);
+
+	const currentActivationStatus = {
+		[STATUS_TAG_TYPE_NAMES.active]: {
+			buttonLink: (
+				<div className="d-flex flex-column">
+					<a
+						className="font-weight-semi-bold m-0 p-0 text-brand-primary text-paragraph"
+						href={`https://${lxcEnvironment?.projectId}.lxc.liferay.com`}
+						rel="noopener noreferrer"
+						target="_blank"
+					>
+						{i18n.translate('go-to-liferay-experience-cloud')}
+
+						<ClayIcon className="ml-1" symbol="order-arrow-right" />
+					</a>
+
+					<a
+						className="font-weight-semi-bold m-0 p-0 text-brand-primary text-paragraph"
+						href={`https://${lxcEnvironment?.projectId}-uat.lxc.liferay.com`}
+						rel="noopener noreferrer"
+						target="_blank"
+					>
+						{i18n.translate('go-to-uat')}
+
+						<ClayIcon className="ml-1" symbol="order-arrow-right" />
+					</a>
+
+					<a
+						className="font-weight-semi-bold m-0 p-0 text-brand-primary text-paragraph"
+						href={`https://analytics.liferay.com/workspace/${project.acWorkspaceGroupId}/sites`}
+						rel="noopener noreferrer"
+						target="_blank"
+					>
+						{i18n.translate('go-to-analytics-cloud-workspace')}
+
+						<ClayIcon className="ml-1" symbol="order-arrow-right" />
+					</a>
+				</div>
+			),
+			id: STATUS_TAG_TYPES.active,
+			subtitle: i18n.translate(
+				'your-experience-cloud-environments-are-ready'
+			),
+			title: i18n.translate('liferay-experience-cloud-activation'),
+		},
+		[STATUS_TAG_TYPE_NAMES.inProgress]: {
+			dropdownIcon: userAccount.isStaff && userAccount.isProvisioning && (
+				<ButtonDropDown
+					align={Align.BottomRight}
+					customDropDownButton={
+						<ButtonWithIcon
+							displayType="null"
+							small
+							symbol="caret-bottom"
+						/>
+					}
+					items={[
+						{
+							label: i18n.translate('set-to-active'),
+						},
+					]}
+					menuElementAttrs={{
+						className: 'p-0 cp-activation-key-icon rounded-xs',
+					}}
+				/>
+			),
+			id: STATUS_TAG_TYPES.inProgress,
+			subtitle: i18n.translate(
+				'your-experience-cloud-project-is-being-set-up-and-will-be-available-soon'
+			),
+			title: i18n.translate('liferay-experience-cloud-activation'),
+		},
+		[STATUS_TAG_TYPE_NAMES.notActivated]: {
+			buttonLink: userAccount.isAdmin && (
+				<Button
+					appendIcon="order-arrow-right"
+					className="btn btn-link font-weight-semi-bold p-0 text-brand-primary text-paragraph"
+					displayType="link"
+					onClick={() => setIsVisibleSetupLxcModal(true)}
+				>
+					{i18n.translate('finish-activation')}
+				</Button>
+			),
+			id: STATUS_TAG_TYPES.notActivated,
+			subtitle: i18n.translate(
+				'almost-there-setup-experience-cloud-by-finishing-the-activation-form'
+			),
+			title: i18n.translate('liferay-experience-cloud-activation'),
+		},
+	};
+
+	const activationStatus =
+		currentActivationStatus[
+			lxcStatusActivation || STATUS_TAG_TYPE_NAMES.notActivated
+		];
```

Comment:
Also, maybe we can do this outside this hook

Reply:
Ok, done!